config/config.go

```diff
@@ -980,6 +980,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.pubnative.endpoint", "http://dsp.pubnative.net/bid/v1/request")
 	v.SetDefault("adapters.pulsepoint.endpoint", "http://bid.contextweb.com/header/s/ortb/prebid-s2s")
 	v.SetDefault("adapters.revcontent.endpoint", "https://trends.revcontent.com/rtb")
+	v.SetDefault("adapters.revcontent.disabled", "true")
```

Comment:
The second parameter here should be a boolean instead a string.

Reply:
Thanks. Sorry about that.

Reply:
No worries. Thanks for fixing it. There's just a pesky merge conflict now.

Reply:
Is that for me to fix, or you?

Reply:
That's for you to fix. Please merge in the master branch and resolve the conflict. Looks like a super minor issue with the user sync tests.